src/client/components/AdminPanel/AdminListings.js

```diff
@@ -35,18 +40,26 @@ export const AdminListings = ({ mediaType }) => {
   };
 
   const retrieveListings = () => {
-    ltikPromise.then(ltik => {
-      axios
-        .get(
-          `/api/medias/${
-            constants.mediaTypeMap.get(mediaType).api
-          }/alllistings?ltik=${ltik}`,
-          {
-            params: { term: recentlySearchedTerm },
-          }
-        )
-        .then(res => setMediaListings(res.data));
-    });
+    const ltik = getLtik();
+    axios
+      .get(
+        `/api/medias/${
+          constants.mediaTypeMap.get(mediaType).api
+        }/alllistings?ltik=${ltik}`,
+        {
+          params: { term: recentlySearchedTerm },
+        }
+      )
+      .then(res => {
+        setMediaListings(res.data);
+        setError(null);
+      })
+      .catch(err => {
+        setError({
+          err,
+          msg: 'Something went wrong when retrieving bruincast listings...',
```

Comment:
Really small thing, but Bruincast should be capitalized to keep it consistent with the rest of the UI?